src/app/clusters/level-control/level-control.cpp

```diff
@@ -408,8 +410,8 @@ bool emberAfLevelControlClusterMoveToLevelCallback(app::CommandHandler * command
     auto & optionsMask     = commandData.optionsMask;
     auto & optionsOverride = commandData.optionsOverride;
 
-    emberAfLevelControlClusterPrintln("%pMOVE_TO_LEVEL %x %2x %x %x", "RX level-control:", level, transitionTime, optionsMask,
-                                      optionsOverride);
+    emberAfLevelControlClusterPrintln("%pMOVE_TO_LEVEL %x %2x %x %x", "RX level-control:", level, transitionTime.Value(),
```

Comment:
This is going to crash if the payload has a null transitionTime.

We should make sure (maybe as a followup) that we have YAML tests exercising these nullable things being null.

Reply:
fixed